src/lib/components/comparisonTable/components/Row/style.module.scss

```diff
@@ -8,7 +8,7 @@ $cell-min-width: var(--cellWidth, 256px);
 
   & > div {
     width: var(--tableWidth);
-    max-width: var(--tableWidth);
+    max-width: calc(100vw - 32px);
```

Comment:
One question, what is the 32px here? is it some padding?
It may be good to add a comment explaining.